pkgs/applications/audio/feishin/default.nix

```
@@ -123,8 +151,11 @@ buildNpmPackage {
     changelog = "https://github.com/jeffvli/feishin/releases/tag/v${version}";
     sourceProvenance = with sourceTypes; [ fromSource ];
     license = licenses.gpl3Plus;
-    platforms = platforms.unix;
+    inherit (electron.meta) platforms;
```

```suggestion
    platforms = platforms.unix;
```

Looks like this caused the `8.has: clean-up` label. I guess `electron.meta` doesn't contain darwin when evaluated on `x86_64-linux`.

---

@wegank Oh, interesting, thank you for catching that; I was trying to be consistent with other `electron` apps I'd seen.

Would you mind explaining why? Doesn't https://github.com/NixOS/nixpkgs/blob/7fd0e9b4456feebd79becd28aaa967343e49e74b/pkgs/development/tools/electron/binary/generic.nix#L41-L43 imply both `"x86_64-darwin"` and `"aarch64-darwin"` would be included?

I navigated the `platforms.*` code
https://github.com/NixOS/nixpkgs/blob/7fd0e9b4456feebd79becd28aaa967343e49e74b/lib/systems/doubles.nix#L110
https://github.com/NixOS/nixpkgs/blob/7fd0e9b4456feebd79becd28aaa967343e49e74b/lib/systems/inspect.nix#L93-L94
https://github.com/NixOS/nixpkgs/blob/3caeeb35f23c6f8ae1fcf2a57fbfa1fca19e5d58/lib/systems/parse.nix#L535
but didn't reach a satisfying conclusion.

I know you probably have too much on your plate, so if there's any resource on this _nuance_, please just link so :)
And again, thank you.